app/views/nsm/work_items/_adjusted.html.erb

```diff
@@ -22,19 +22,26 @@
     </thead>
 
     <tbody class="govuk-table__body app-task-list__items">
+      <% type_changed_records = [] %>
       <% records.each do |work_item| %>
         <tr id="<%= work_item.id %>" class="govuk-table__row app-task-list__item">
           <th class="govuk-table__header" scope="row"><%= work_item.position %></th>
           <th class="govuk-table__header" scope="row">
-            <%=
-              if claim.editable_by?(current_user)
-                link_to(work_item.work_type, edit_nsm_claim_work_item_path(claim_id: claim.id, id: work_item.id), data: { turbo: 'false' })
-              else
-                link_to(work_item.work_type, nsm_claim_work_item_path(claim_id: claim.id, id: work_item.id), data: { turbo: 'false' })
-              end
-            %>
-            <% if work_item.work_type_original.present? %>
-              *
+            <% link = if claim.editable_by?(current_user)
```

Comment:
Would `polymorphic_path` be neater here?

Reply:
polymorphic_path doesn't quite do it in this case, simply because `work_item` is a `Nsm::V1::WorkItem` and without a bunch more tinkering that gets interpreted as `...nsm_v1_work_item...` which isn't a valid route. But `url_for` allows us to pass in an `action` option, so I've used that instead to avoid the if clause.